packages/@react-spectrum/s2/src/Image.tsx

```diff
@@ -310,7 +310,7 @@ export const Image = forwardRef(function Image(props: ImageProps, domRef: Forwar
 
     if (Array.isArray(srcProp)) {
       img = (
-        <picture>
+        <picture className={style({objectFit: 'inherit', objectPosition: 'inherit'})}>
```

Comment:
I assume this is due to lines 176-177?

Reply:
Yeah the previous change to add this picture wrapper broke it. You can see the card images on the Examples page are stretched. Even though `picture` doesn't render anything, it still participates in CSS property inheritance.